lib/rust/shapely/macros/src/derive_entry_point.rs

```diff
@@ -33,7 +33,10 @@ pub fn derive(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
             let output = quote! {
                 #(#attrs)*
                 #[wasm_bindgen]
-                pub #fn_sig #block
+                pub #fn_sig {
+                    init_tracing(WARN);
```

Comment:
I must dig very deep to enable more detailed logging. Could the level be an alias/constant somewhere in the prelude?